logbook-api/src/main/java/org/zalando/logbook/RequestURI.java

```diff
@@ -10,35 +10,47 @@
 import static org.zalando.logbook.RequestURI.Component.QUERY;
 import static org.zalando.logbook.RequestURI.Component.SCHEME;
 
-final class RequestURI {
+public final class RequestURI {
```

Comment:
Open for suggestions how to do this better

Reply:
Why does it need to be public?

Reply:
:ping_pong: :wink: